### PR TITLE
fix: /var/local paths should be under release name

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
## Description
Currently we use `/var/local/io-engine` and `/var/local/local-pv-hostpath/{{ .Release.Name }}`
This is a bit too spread we should bundle it all within a common `/var/local/{{ .Release.Name }}` example:-
`/var/local/{{ .Release.Name }}/{io-engine, localpv-hostpath}`

This PR changes the helm chart to include this.

## Regression
 No

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -386,7 +386,7 @@ etcd:
     # Name of etcd's localpv hostpath storage class.
     name: "mayastor-etcd-localpv"
     # -- Host path where local etcd data is stored in.
-    basePath: "/var/local/localpv-hostpath/{{ .Release.Name }}/etcd"
+    basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/etcd"
```
Comment (tiagolobocastro) [inline chart/values.yaml:389]: Presumably this doesn't have any issues for upgrades as the PV's were already created anyway, but new PV's will have this new path?
@niladrih we probably need to ensure this gets updated?

File: chart/values.yaml [hunk 2]
```diff
@@ -486,7 +486,7 @@ loki-stack:
     # Name of loki's localpv hostpath storage class.
     name: "mayastor-loki-localpv"
     # -- Host path where local etcd data is stored in.
-    basePath: "/var/local/localpv-hostpath/{{ .Release.Name }}/loki"
+    basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath//loki"
```
[the remark this answers is not in the document]
Comment (Abhinandan-Purkait) [reply chart/values.yaml]: Damn. Thanks for pointing out. Done

File: chart/values.yaml (hunk 1 above)
Comment (niladrih) [reply chart/values.yaml:389]: Hmmm, so this should not get updated at upgrade-time, correct?

Comment (Abhinandan-Purkait) [reply chart/values.yaml:389]: Yes, this can potentially patch the io-engine ds with the new path but we don't want that.

Comment (Abhinandan-Purkait) [reply chart/values.yaml:389]: Right. But what about the mayastor-io-engine ds, if that gets patched with new path would it be a problem ?

Comment (tiagolobocastro) [reply chart/values.yaml:389]: As things are today no I don't think so.

Comment (niladrih) [reply chart/values.yaml:389]: Alright then, then I'm going with no-change because both no-change and change seem harmless.

**Review by niladrih — APPROVED** [review]
_“What's in a {{ .Release.Name }}? That which we call a rose by any other name would smell just as sweet.”_ - William Shakespeare

**Review by tiagolobocastro — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**